conda_build/metadata.py

```diff
@@ -256,7 +266,8 @@ def _git_clean(source_meta):
               'no_link', 'binary_relocation', 'script', 'noarch_python',
               'has_prefix_files', 'binary_has_prefix_files', 'script_env',
               'detect_binary_files_with_prefix', 'rpaths',
-              'always_include_files', 'skip', 'msvc_compiler'],
+              'always_include_files', 'skip', 'msvc_compiler',
+              'pin_depends'],
```

Comment:
Add in comment here that `pin_depends` is an alpha feature with names/api/etc subject to change.